hbw/app/javascript/packs/hbw/components/form/string.js.jsx

```diff
@@ -61,7 +61,8 @@ modulejs.define('HBWFormString', ['React'], (React) => {
           <input {...opts}
                  ref={(i) => { this.input = i; }}
                  className={`form-control ${!this.state.valid && ' invalid'}`}
-                 value={this.state.visualValue} />
+                 value={this.state.visualValue}
+                 disabled={this.props.disabled} />
```

Comment:
Don't we want to make it `readOnly` to allow input content selection with cursor?

Reply:
Good point